plugins/gradle/src/index.ts

```diff
@@ -103,6 +106,7 @@ export default class GradleReleasePluginPlugin implements IPlugin {
   private readonly updateGradleVersion = async (
     version: string,
     commitMsg?: string,
+    commit = true,
```

Comment:
@sentony93 Adding a new param in the middle breaks existing callers. See [line #356](https://github.com/intuit/auto/pull/1842/files#diff-0b8b94d428b4efb389ba5afae6b74e1824b87e6f3c1bad0fbf442a13c605618dR356). This breaks the actual release since it no longer commits the prepared version. You should reorder the new argument to the end to ensure that you don't break existing call-sites.